codalab/worker/worker_run_state.py

```diff
@@ -290,8 +291,9 @@ def mount_dependency(dependency, shared_file_system):
         if not dependencies_ready:
             status_message = status_messages.pop()
             if status_messages:
-                status_message += "(and downloading %d other dependencies and docker images)" % len(
-                    status_messages
+                status_message += (
+                    "\n(and downloading %d other dependencies and docker images)"
```

Comment:
Do we need this newline?

Reply:
I think it looks weird without it.

Reply:
Ok -- the screenshot on this PR seems to not have the newline -- can you change the screenshot to the latest one so I can see how it looks like with the newline added?

![image](https://user-images.githubusercontent.com/1689183/111198611-2064d600-8596-11eb-88b7-9425f0826d08.png)